azure-pipelines.yml

```diff
@@ -36,6 +36,18 @@ variables:
   modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
 
 steps:
+- checkout: self
+  persistCredentials: true
+  clean: true
+
+- task: ShellScript@2
+  displayName: Validate GitOps pipeline
+  inputs:
+    scriptPath: gitops/azure-devops/build.sh
```

Comment:
@samiyaakhtar which Bedrock repository branch does the `gitops/azure-devops/build.sh` file come from?

Reply:
It would come from the branch that is being pull requested, the checkout step ensures the tests run on the branch that's being requested to be merged